modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesMessageMetadata.java

```diff
@@ -55,6 +55,10 @@
      * The timestamp at which the message arrived at the first gates of the broker.
      */
     long arrivalTime;
+    /**
+     * Whether Expiration value is set for the message
+     */
+    boolean isExpirationSet;
```

Comment:
best practice  is to have getters/setters to align with variable name. ( e.g.  boolean isExpirationDefined)